### PR TITLE
skip three failing tests

### PR DESCRIPTION
**Pull request by devoncarew**
- skip ~two~ three failing tests
- related to https://github.com/flutter/devtools/issues/1773 and https://github.com/flutter/devtools/issues/1778

### CODE REVIEW

**Comment by devoncarew** [comment]
`devtools_app/test/memory_service_test.dart` is now failing...

**Comment by devoncarew** [comment]
I'll take a look at the failures and try to get these tests back on line shortly.

File: packages/devtools_app/test/timeline_controller_test.dart [hunk 1]
```diff
@@ -2,18 +2,21 @@
...
+  // TODO(devoncarew): Skip the timeline_controller_test.dart tests (#1778).
+  // ignore: dead_code
+  if (false) {
+    final FlutterTestEnvironment env = FlutterTestEnvironment(
+      const FlutterRunConfiguration(withDebugger: true),
+    );
 
-  await runTimelineControllerTests(env);
+    await runTimelineControllerTests(env);
+  }
```
Comment (kenzieschmoll) [inline packages/devtools_app/test/timeline_controller_test.dart:21]: you may also need to make this change to devtools_app/test/flutter/timeline_controller_test

Comment (devoncarew) [reply packages/devtools_app/test/timeline_controller_test.dart:21]: Cool; I'll add that test to the exclusion if we see a flake from it. I'm hoping these tests won't have to be disabled long.

**Pull request merged**